include/ros1_bridge/factory.hpp

```diff
@@ -125,23 +138,25 @@ class Factory : public FactoryInterface
     auto ros2_msg = std::make_shared<ROS2_T>();
     convert_1_to_2(*ros1_msg, *ros2_msg);
     RCUTILS_LOG_INFO_ONCE_NAMED(
-      "ros1_bridge", "Passing message from ROS 1 "
-      _ros1_bridge_type_to_name(ROS1_T) " to ROS 2 "
-      _ros1_bridge_type_to_name(ROS2_T) " (showing msg only once per type)");
+      "ros1_bridge",
+      "Passing message from ROS 1 %s to ROS 2 %s (showing msg only once per type)",
+      ros1_type_name.c_str(), ros2_type_name.c_str());
```

Comment:
Would be nice if this included the topic it was doing this on.

Reply:
I was only aiming to fix the bug with this. Please feel free to extend the patch to add additional information.

Reply:
Should we wait for this to be updated or merge it as-is and make additional improvements in a separate PR?

Reply:
You can merge it as is, I’ll open another pr if I have a minute to add the topic name.

Reply:
You can merge it as is, I’ll open another pr if I have a minute to add the topic name.